source/common/http/http1/BUILD

```diff
@@ -13,6 +13,7 @@ envoy_cc_library(
     srcs = ["codec_impl.cc"],
     hdrs = ["codec_impl.h"],
     external_deps = ["http_parser"],
+    copts = ["-Wno-old-style-cast"],
```

Comment:
Not crazy about this. Why can't we pull in http-parser via -isystem?

Reply:
I didn't realize `-isystem` silenced these warnings. Changed to use angle-bracket includes for http_parser and backward.